src/pages/Todo2.jsx

```diff
@@ -0,0 +1,212 @@
+import { useState, useEffect } from "react";
+
+export default function Todo() {
+    const [searchText, setsearchText] = useState("");
+    const [tasks, settasks] = useState([
+      {
+        id: 1,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        status: "completed",
+        name: "Coding react",
+      },
+      {
+        id: 2,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        status: "pending",
+        name: "Prepare Nextjs",
+      },
+    ]);
+  
+    const [filteredResults, setfilteredResults] = useState([]);
+  
+    useEffect(() => {
+      if (searchText) {
+        const res = tasks.filter((e) =>
```

Comment:
You can keep the naming consistent and self-descriptive, for this case `e` does not explain much by itself
```suggestion
        const res = tasks.filter((task) =>
```